packs/yammer/actions/post_message.py

```diff
@@ -0,0 +1,13 @@
+from lib.actions import YammerAction
+
+__all__ = [
+    'PostMessageAction'
+]
+
+
+class PostMessageAction(YammerAction):
+    def run(self, message, group_id, topics, replied_to_id):
```

Comment:
Minor thing - you can also default all the optional arguments to None (topics, group_id, replied_to_id).